content/en/user-guide/integrations/testcontainers/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Testcontainers"
+tags: [
+  "testcontainers",
+  "test",
+  "java",
+  "jvm",
+  "kotlin",
+  "go",
+  "dotnet"
+]
+categories: ["LocalStack Community", "LocalStack Pro"]
+date: 2023-02-11
+weight: 5
+description: >
+  Use Testcontainers with LocalStack
+aliases:
+  - /integrations/testcontainers/
+---
+
+<img src="testcontainers-logo.svg" width="600px" alt="Testcontainers logo"><br />
+
+## Overview
+
+[Testcontainers](https://www.testcontainers.com/) is a library that helps you to run your
+tests against real dependencies.
+
+In this guide, you will learn how to use [Testcontainers](https://www.testcontainers.com/) 
+with LocalStack.
+
+## Covered Topics
+
+* [Testcontainers for .NET](#testcontainers-for-net)
+* [Testcontainers for Go](#testcontainers-for-go)
+* [Testcontainers for Java](#testcontainers-for-java)
+
+### Testcontainers for .NET
+
```

Comment:
~~~suggestion
### Testcontainers for .NET

The current released version of Testcontainers for .NET does not include a LocalStack module. However, developers who want to use Testcontainers with LocalStack should not be discouraged, as the Testcontainers team has already developed a LocalStack module that will be included in the next release. In the meantime, developers can still use Testcontainers' generic builder to create a LocalStack container manually.

```console
dotnet add package Testcontainers --version 2.4.0
```

In order to use LocalStack in your tests, you need to build a `LocalStackContainer` instance and start it

```csharp
const string localStackImage = "localstack/localstack:1.4.0";

const ushort localStackPort = 4566;

var localStackContainer = new ContainerBuilder()
    .WithImage(localStackImage)
    .WithPortBinding(localStackPort, true)
    .WithWaitStrategy(Wait.ForUnixContainer().UntilHttpRequestIsSucceeded(request => request.ForPath("/_localstack/health").ForPort(localStackPort)))
    .Build();

await localStackContainer.StartAsync()
    .ConfigureAwait(false);
```

~~~

Reply:
I submitted a PR restructuring the page layout to leverage tabs, as in https://docs.localstack.cloud/user-guide/integrations/copilot/ You probably want to take a look there: https://github.com/eddumelendez/localstack-docs/pull/1

Reply:
But how should I add the explanation to it? The `tabpane` seems only to cover the code. As code comment?

Reply:
It's possible to add comments to the tab code. In any case, you could add a note before the tabs.

Reply:
Updated the `tc-go` branch.